### PR TITLE
result page

### PR DESCRIPTION
**Pull request by yogesh-**
- user can see total score
- navigate to the see correct answers
image

image

### CODE REVIEW

**Comment by Husain01** [comment]
The buttons in the quiz can be styled a little better and given a definite width and height to make them uniform.

**Review by Kothalkarpriya — APPROVED** [review]
Good to go

**Comment by Husain01** [comment]
On the Results page even though I have wrong answers the answer is displayed in green, check the logic of putting the classes for the results.

**Comment by Husain01** [comment]
Try storing an array of the selectedOptions and then compare those with the right answer. As you are just mapping over the array of objects from the db the results are shown as green since your function to check is true always.

**Review by Husain01 — CHANGES_REQUESTED** [review]
Try storing an array of the selectedOptions and then compare those with the right answer. As you are just mapping over the array of objects from the db the results are shown as green since your function to check is true always.

File: src/context/resultContext/resultContext.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const ResultContext = createContext();
+
+const ResultProvider = ({ children }) => {
+  const [resultState, resultDispatch] = useReducer(ResultReducer, {
+    userQuest: "",
+    userAns: "",
+    ansFlag: "",
+    userData: [],
+
+    // // new data
```
Comment (parulgupta2001) [inline src/context/resultContext/resultContext.jsx:13]: remove commented code

File: src/pages/result/result.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  flex-wrap: wrap;
+  margin-top: 5rem;
+}
+
+.answer button:hover {
+  background: none;
+}
+
+.correct {
+  background-color: greenyellow;
```
Comment (parulgupta2001) [inline src/pages/result/result.css:19]: use variable for color

**Review by parulgupta2001 — APPROVED** [review]
Some suggestions are given, otherwise code is clean and understandable.

**Comment by yogesh-** [comment]
> Try storing an array of the selectedOptions and then compare those with the right answer. As you are just mapping over the array of objects from the db the results are shown as green since your function to check is true always.

At the moment i am just displaying the correct answers from the db. user selected option will be given in the next iteration.

**Pull request merged**